test/local/geodb.js

```diff
@@ -5,6 +5,7 @@
 'use strict'
 
 const { assert } = require('chai')
+const ipLocation = require('../ip-location')
```

Comment:
As a pure naming nit, calling this something like "knownIpLocation" might make the tests read a little clearer, but that's a very small nit